collections.yaml

```diff
@@ -0,0 +1,54 @@
+allOf:
+  - type: object
+  proprties:
+    id:
+      type: number
+      description: |-
+        Identifier for the collection.
+    author:
+      type: string
+      description: |-
+        Author or authors of the collection the image/annotation belongs to.
+    title:
+      type: string
+      description: |-
+        Title of the collection.
+    year:
+      type: number
+      minimum: 1900
+      maximum: 3000
+      description: |-
+        Collection publication year.
+    identifier:
+      type: string
+      description: |-
+        A DOI or other identifier used to find the collection.
+    rights:
+      type: string
+      description: |-
+        Copyright status of the collection.
+        Further detail available in the licences object.
+    accrual_policy:
+      type: string
+      enum:
+        - closed
+        - open
+      description: |-
+        A description of the accrual policy for this collection.
+        Open collections can be contributed to, and closed collections cannot be.
+
+  - type: object
```

Comment:
Is this to allow for a Collection to either be a DCMI-complete Collection definition, or to be a "subset of a collection" by reference?

If so:
* allOf above should be anyOf, since it's an either/or situation.
* I'm not sure why we want `image_id` here.
* Why do you prefer this to having `collection_memberships`?